### PR TITLE
Buffs Alien Queens

### PR DESCRIPTION
**Pull request by GDNgit**
## What Does This PR Do
Buffs the alien queen
Ups their disarm damage to 60 from 30
Ups their slash damage to 30 from 20 (Hal helped me with this) 
Decreases their movement delay from 3 to 1
Prevents alien queens from being pulled/pushed
(all of these changes have been mirrored to empresses as well) 
## Why It's Good For The Game
Currently, alien queen is fucking boring, everyone wants someone else to go queen so they don't need to, alien queens also suck at defending themselves. Alien queens are now a force not to be fucked with, it stamcrits you in two strikes (provided you don't have an item in hand), and deals esword level damage.
The decrease to movement speed will make basic movement less of a slog, while removing the reliance on getting a hunter to pull you
Alien queens can no longer be pushed/pulled, this prevents people from pushing queens out of the nest, and fast people from grabbing and running off with them.
This should hopefully make it actually fun to play.

## Testing
Did funny stuff with a skrell and some xenos
## Changelog
:cl: GDN , hal9000
tweak: Alien queens can no longer be pulled/pushed
tweak: Alien queens deal 10 more brute damage
tweak: Alien queens deal 30 more stamina damage
tweak: Alien queens move faster
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/alien/humanoid/queen.dm [hunk 1]
```diff
@@ -9,6 +9,11 @@
 	large = 1
 	ventcrawler = 0
 	pressure_resistance = 200 //Because big, stompy xenos should not be blown around like paper.
+	move_resist = MOVE_FORCE_STRONG //Yes, queenos is huge and heavy
+	alien_disarm_damage = 60 //Queens do higher disarm stamina damage than normal aliens
+	alien_slash_damage = 30 //Queens do higher slashing damage to people
+	var/alien_queen_movement_delay = 1 //This represents a movement delay of 1, or roughly 80% the movement speed of a normal carbon mob
```
Comment (hal9000PR) [inline code/modules/mob/living/carbon/alien/humanoid/queen.dm:15]: this should probably be another `/mob/living/carbon/alien/humanoid` level var

File: code/modules/mob/living/carbon/human/human_defense.dm [hunk 2]
```diff
@@ -625,19 +625,14 @@ emp_act
...
-				return 0
 			var/obj/item/organ/external/affecting = get_organ(ran_zone(M.zone_selected))
 			var/armor_block = run_armor_check(affecting, MELEE, armour_penetration_flat = 10)
 
 			playsound(loc, 'sound/weapons/slice.ogg', 25, TRUE, -1)
 			visible_message("<span class='danger'>[M] has slashed at [src]!</span>", \
  				"<span class='userdanger'>[M] has slashed at [src]!</span>")
 
-			apply_damage(damage, BRUTE, affecting, armor_block)
+			apply_damage(adjustBruteLoss(M.alien_slash_damage), BRUTE, affecting, armor_block)
```
Comment (hal9000PR) [inline code/modules/mob/living/carbon/human/human_defense.dm:635]:
```suggestion
			apply_damage(M.alien_slash_damage, BRUTE, affecting, armor_block)
```
this is what I meant

**Review by hal9000PR — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/alien/humanoid/queen.dm (hunk 1 above)
Comment (GDNgit) [reply code/modules/mob/living/carbon/alien/humanoid/queen.dm]: Addressed

File: code/modules/mob/living/carbon/human/human_defense.dm (hunk 2 above)
Comment (GDNgit) [reply code/modules/mob/living/carbon/human/human_defense.dm]: Addressed

File: code/modules/mob/living/carbon/alien/humanoid/caste/hunter.dm [hunk 3]
```diff
@@ -16,7 +17,7 @@
 	. += /obj/item/organ/internal/xenos/plasmavessel/hunter
 
 /mob/living/carbon/alien/humanoid/hunter/movement_delay()
-	. = -1		//hunters are sanic
+	. = alien_movement_delay		//hunters are sanic
 	. += ..()	//but they still need to slow down on stun
```
Comment (hal9000PR) [inline code/modules/mob/living/carbon/alien/humanoid/caste/hunter.dm:21]: this should be on the base `/mob/living/carbon/alien/humanoid` level. no need to have the same copy paste code on every alien caste

Comment (GDNgit) [reply code/modules/mob/living/carbon/alien/humanoid/caste/hunter.dm]: Addressed

**Review by hal9000PR — APPROVED** [review]

**Comment by Dafrek** [comment]
I might be missing something but, despite this PR having been merged, it seems like at least some of the intended changes from this PR have not actually changed the xeno queen. I've played as a queen twice on the paradise server after this change, once an admin spawned one and once a regularly spawned one and I noticed it still seems to take 4 disarms to stam-crit people.

**Pull request merged**